prow/config.yaml

```diff
@@ -36,6 +36,8 @@ deck:
   tide_update_period: 1s
   hidden_repos:
   - kubernetes-security
+  default_org: kubernetes
```

Comment:
You should not require that the same data have two sources of truth in the config

Reply:
I meant to undo this change earlier